.github/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 2. AIRAC (2014) - Updated EGHE Runway Headings - thanks to @AleksMax (Aleks Nieszczerzewski)
 3. Enhancement - Added EGBB/EGNX Buffer Zone - thanks to @AleksMax (Aleks Nieszczerzewski)
 4. Bug - Corrected Gatwick (EGKK) Stands - thanks to @WillShaw1100 (William Shaw)
+Procedure Change - Fixes #3261 - Amended EGJB Positions
```

Comment:
it's usual to add 'thanks to @[github username] (your name)' - you can remain anonymous if you want, but didn't want you missing out on credit 😄

Reply:
Hah, thanks. I've updated that now.